BlueprintUI/Sources/Layout/Builder/GridRowChild.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+// MARK: - child modeling -
+extension GridRow {
+    /// A child of a `GridRow`.
+    public struct Child {
```

Comment:
Same request here re: moving into the parent file. The Child type is an inseparable part of the implementation of GridRow.

Reply:
Bump here too

Reply:
Done.